### PR TITLE
make the fileviewer take up the full size of the containing flexbox

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**
Needs this:
https://github.com/Brightspace/activity-viewer/pull/61

### CODE REVIEW

**Comment by omsmith** [comment]
Don't think we should have that here. The consumer should be assigning the flexiness to the outermost element

File: sample/app.js [hunk 1]
```diff
@@ -45,7 +45,8 @@ var Main = React.createClass({
 				progressCallback={this.logProgress}
 				src={'files/' + this.state.file.src}
 				srcdownload={'files/' + this.state.file.src}
-				locale={this.state.locale} /> : null;
+				locale={this.state.locale}
+				className='file-viewer-sample' /> : null;
```
Comment (d2l-joelp) [inline sample/app.js]: I'm not seeing `file-viewer-sample` used anywhere.

File: sample/app.scss [hunk 2]
```diff
@@ -24,3 +24,13 @@ hr {
 	border-bottom: none;
 	margin: 0;
 }
+
+.vui-fileviewer {
```
Comment (d2l-joelp) [inline sample/app.scss]: I don't see `vui-fileviewer` or `#placeholder` used anywhere.

File: sample/app.js (hunk 1 above)
Comment (jstefaniuk-d2l) [inline sample/app.js]: Whoops, good catch, removing now

File: sample/app.scss (hunk 2 above)
Comment (jstefaniuk-d2l) [inline sample/app.scss]: placeholder is an id in index.html, and .vui-fileviewer is the fileviewer's class (set in fileViewerResolved.js)

**Comment by CodeBaboon** [comment]
LGTM

**Pull request merged**